cmd/plugin/README.md

```diff
@@ -0,0 +1,23 @@
+# apisix-ingress-controller plugin
+
+The apisix-ingress-controoler plugin can quikly show the apisix routes and upstreams in kubernetes cluster.
+
+# Quikly start 
+
+```shell
+git clone http://github.com/apache/apisix-ingress-controller 
+cd cmd/plugin
+sh install-plugin.sh
+```
+
+```shell 
+# Show your routes in kubernetes.
+kubectl apisix-ingress-controller routes -n <APISIX_INGRESS_NS>
```

Comment:
IMHO `apisix-ingress-controller` is too long, maybe `ingress-apisix` is more concise.

Reply:
no, I think our project is `apache/apisix-ingress-controller` , so I think used it.